content/_img-sources/ledger-object-ids.ja.uxf

```diff
@@ -0,0 +1,587 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<diagram program="umlet" version="14.2">
```

Comment:
Reviewers can skip these `.uxf` files; these are just the source files (made in UMLet) for the [SVG diagrams](https://github.com/ripple/xrpl-dev-portal/blob/332a66e57710f8e70291c0b2eda96c1d2e287d7f/img/ledger-object-ids.svg).